### ISSUE TITLE
help text for debug-level CLI option 

### ISSUE THREAD

**Issue by jrhea** [sigp/lighthouse#997]
## Description

I was browsing through the lighthouse code and it looks like the `debug-level` cli option might have the wrong help text:

**code:**
https://github.com/sigp/lighthouse/blob/869b0621d6f51e98473bb67cec0446d3623b8957/lighthouse/src/main.rs#L59-L62

**cli:**
```
$ ./lighthouse --help
Lighthouse 0.2.0
Sigma Prime <contact@sigmaprime.io>
Ethereum 2.0 client by Sigma Prime. Provides a full-featured beacon node, a validator client and utilities for managing
validator accounts.

USAGE:
    lighthouse [OPTIONS] [SUBCOMMAND]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -d, --datadir <DIR>          Data directory for lighthouse keys and databases.
        --debug-level <LEVEL>    The title of the spec constants for chain config. [default: info]  [possible values:
                                 info, debug, trace, warn, error, crit]
        --log-format <FORMAT>    Specifies the format used for logging. [possible values: JSON]
        --logfile <FILE>         File path where output will be written. Default file logging format is JSON.
    -s, --spec <TITLE>           Specifies the default eth2 spec type. [default: mainnet]  [possible values: mainnet,
                                 minimal, interop]

SUBCOMMANDS:
    account_manager     Utilities for generating and managing Ethereum 2.0 accounts. [aliases: a, am, account,
                        account_manager]
    beacon_node         The primary component which connects to the Ethereum 2.0 P2P network and downloads, verifies
                        and stores blocks. Provides a HTTP API for querying the beacon chain and publishing messages
                        to the network. [aliases: b, bn, beacon]
    help                Prints this message or the help of the given subcommand(s)
    validator_client    When connected to a beacon node, performs the duties of a staked validator (e.g., proposing
                        blocks and attestations). [aliases: v, vc, validator]
```

**Comment by AgeManning** [comment]
Yep. looks wrong. Will update. Thanks @jrhea

**Comment by AgeManning** [comment]
Resolved in #999
